src/systems/filecoin_markets/storage_market/storage_market_flow.mmd

```diff
@@ -0,0 +1,91 @@
+sequenceDiagram
```

Comment:
This is the most beautiful mmd diag in the spec, and the most useful imo!!!!

Adding these rectangles so we can indicate larger context on sequence (eg in the dea-payment flow etc) really helps